internal/infraenv/infraenv_test.go

```diff
@@ -64,29 +65,50 @@ var _ = Describe("Delete inactive infraenvs", func() {
 		ctx          = context.Background()
 		db           *gorm.DB
 		state        API
-		infraEnv     common.InfraEnv
 		dbName       string
 		mockS3Client *s3wrapper.MockAPI
 	)
 
+	const emptyClusterID = ""
+
+	var (
+		defaultTime           time.Time = time.Date(2012, time.December, 21, 5, 1, 2, 6, time.UTC)
```

Comment:
[Nice](https://en.wikipedia.org/wiki/2012_phenomenon). Intentional or happy accident?

Reply:
I thought the `5, 1, 2, 6` would remove any doubt